.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -145,7 +167,7 @@ body:
       description: What device you are experiencing this problem on? Specify full device name along with the version of the operating system it's running.
       placeholder: ex. Samsung Galaxy A22 (Android 12)
 
-  - type: dropdown
+  - type: checkboxes
```

Comment:
There's a particular reason why we chose to use dropdown here &ndash; each checkbox inside issue description is treated like a task and shown like this in the issues list:

<img width="91" alt="Zrzut ekranu 2023-08-1 o 00 00 40" src="https://github.com/software-mansion/react-native-reanimated/assets/20516055/af91e873-3c40-4664-ac45-b5f6775195ba">

Reply:
Yes, but github is going to fix this soon: https://github.com/orgs/community/discussions/5197#discussioncomment-5966864